### PR TITLE
WIP : 1115/i18n : Upgrade ngx-moment + Fix Function expressions are not sup…

### PR DESCRIPTION
**Pull request by cedricjimenezst**
…ported in decorators

### CODE REVIEW

**Review by francoisno — CHANGES_REQUESTED** [review]
Hi @cedricjimenezst thanks a lot for the PR, I will test it in the next days!
(And I'll be glad to work on #1115 with you if you're interested.)

Concerning this PR, it is no big deal but could you refactor it slightly (branch name, commit message, signed commit) to fit project conventions please? (https://github.com/theopenconversationkit/tock/blob/master/CONTRIBUTING.md)
Please let me know if you have questions.

**Comment by vsct-jburet** [comment]
@francoisno can I merge this MR ?

File: nlp/admin/web/src/app/quality-nlp/quality.module.ts [hunk 1]
```diff
@@ -36,6 +36,11 @@ import {NbThemeModule, NbRouteTabsetModule, NbCardModule, NbButtonModule, NbSele
 import {ThemeModule} from "../theme/theme.module";
 import { NgxEchartsModule } from 'ngx-echarts';
 import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
+import {ChartsModule} from "ng2-charts";
```
Comment (francoisno) [inline nlp/admin/web/src/app/quality-nlp/quality.module.ts:39]: This import/module breaks the build (at least after rebasing from master).

**Review by francoisno — CHANGES_REQUESTED** [review]
Once the `ng2-charts` import is removed from NLP Admin, every works well.
I will rebase, amend the commit and merge manually the PR.
Thx again.

**Comment by francoisno** [comment]
Merged manually.

**Pull request closed without merging**